### DISCUSSION TITLE
Remove Backlinks

### DISCUSSION THREAD

**Discussion by GoRoX98** [anyproto/contributors#236] [Q&A] [answered]
Hello!
We have several objects with unnecessary "backlinks" in our project. We dont see a fuctional for remove it.
Can u help with it?

**Comment by filip-anytype** [comment]
Would you be able to share a screenshot of these backlinks? Without more info, it would be really hard to figure out what the exact issue is.

**Reply by GoRoX98** [reply]
First screenshot highlighted backlink
Second screenshot - its highlighted page from first scrn, but also have backlink to page from first scrn
<details>
<summary>Screenshots</summary>

![img](https://github.com/user-attachments/assets/3fe56ec9-a4bc-4e3e-9bdd-3860ae677321)
![image](https://github.com/user-attachments/assets/3a916c91-6dd4-4142-afa6-e36635e31ed0)

</details>

I would like to del backlink to main page from page on the second screenshot.

**Reply by filip-anytype** [reply]
> First screenshot highlighted backlink Second screenshot - its highlighted page from first scrn, but also have backlink to page from first scrn
> Screenshots
> 
> I would like to del backlink to main page from page on the second screenshot.

Hey @GoRoX98, just from these screenshots, it's not really clear how you've linked these two objects together. You can [use the graph view](https://doc.anytype.io/anytype-docs/basics/linking-objects#relations-on-the-graph) to check why these two objects are linked together, and afterwards, you can either remove them from the relevant relations, or remove the link blocks in one of the two objects.

**Reply by GoRoX98** [reply]
@filip-anytype 
>You can use the graph view

![image](https://github.com/user-attachments/assets/9672efc4-45ad-4bf7-ba95-7a3edadc3b83)

**Reply by filip-anytype** [reply, marked as answer]
Seems like you've used the Sub-Item relation to connect these two objects together. You can remove these two objects from the relation, and they will no longer appear in the backlinks section nor will they appear connected in the graph.

Here's an example using a different relation:

### Before

![image](https://github.com/user-attachments/assets/761981e8-b664-45a6-bfda-7b12c7ecf190)

![image](https://github.com/user-attachments/assets/42f94ee2-5b80-4d33-8472-df6ca4af62c6)

### After

![image](https://github.com/user-attachments/assets/695abe4f-081d-4d9d-8aea-eafb3e8e2129)

![image](https://github.com/user-attachments/assets/51a3c2f9-b368-4bd6-ac6b-fe0420bcad9a)
